profile/README.md

```diff
@@ -0,0 +1,48 @@
+<div align="center" style="background-color: white;">
+<a href="https://www.transportforthenorth.com/">
+<img src="https://www.transportforthenorth.com/wp-content/themes/tfn-theme/img/logo.svg"
+  alt="Transport for the North logo">
+</a>
+</div>
+
+# About Us
+
+Transport for the North (TfN) are England's first Sub-national Transport Body (STB). We were formed to transform the transport system across the North of England, providing the infrastructure needed to drive economic growth.
+TfN's role is to add strategic value by ensuring that funding and strategy decisions about transport in the North are informed by local knowledge and requirements, more details on our work can be found on our website at [transportforthenorth.com](https://www.transportforthenorth.com/).
+
+# Open Source Analytics
+
+TfN has a focus on openness and transparency and as such many of our analytics tools and processes are shared openly on GitHub for scrutiny and use by others.
+The repositories fall into one of two categories, either internal TfN or common analytical framework.
```

Comment:
```suggestion
TfN has been building analytical tools for business cases since its inception in 2018. We believe that the best way to achieve the highest value for money from these tools is to share them with partners and other public bodies in the UK. Therefore, we share many of our analytics tools and processes on GitHub, allowing for transparency, scrutiny, and use by others.
The repositories fall into one of two categories: internal TfN, or Common Analytical Framework.
```